### PR TITLE
LMB- 370: dct:modified to mandataris, fractie and person

### PR DESCRIPTION
**Pull request by JonasVanHoof**
These entities should have a property dct:modified to keep track of the changes.

## Test
When checking out this PR make sure to also checkout these branches
* **App:** https://github.com/lblod/app-lokaal-mandatenbeheer/pull/115 
* **Form-content-service:** https://github.com/lblod/form-content-service/pull/44

### CODE REVIEW

File: app/models/fractie.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ export default class FractieModel extends Model {
   @attr uri;
   @attr naam;
   @attr('uri-set') generatedFrom;
+  @attr('datetime') modified;
```
Comment (JonasVanHoof) [inline app/models/fractie.js:7]: Should we add a default? 
```js
  @attr('datetime', {
    defaultValue() {
      return new Date();
    },
  })
  modified;

File: app/routes/organen/fracties.js [hunk 2]
```diff
@@ -102,7 +102,7 @@ export default class FractiesRoute extends Route {
   }
 
   @action
-  reloadModel() {
+  async reloadModel() {
```
Comment (ruben-bottu) [inline app/routes/organen/fracties.js:105]: Not sure how useful the async is here if you don't await anything.

**Review by ruben-bottu — APPROVED** [review]

**Review by ruben-bottu — COMMENTED** [review]
Should there maybe also be a dct:modified for creating instances?

**Review by ruben-bottu — CHANGES_REQUESTED** [review]
This feedback applies to app, form-content and frontend:

Newly created instances should also have a dct:modified. This way our data will be more easy to consume for the vendors.

File: app/routes/organen/fracties.js (hunk 2 above)
Comment (JonasVanHoof) [reply app/routes/organen/fracties.js]: Removed it

**Comment by JonasVanHoof** [comment]
> This feedback applies to app, form-content and frontend:
> 
> Newly created instances should also have a dct:modified. This way our data will be more easy to consume for the vendors.

When an entity is created through the store it gets the default value for modified (new Date() at that point in time) but indeed when a post is done through the form-content service this could not be the case. Will check this out. 

Next to the upcoming change in the form-content-service I think we can already merge the current PR's right?

**Review by ruben-bottu — APPROVED** [review]
Approved on the condition that form-content-service also creates dct:modified for new entities

**Pull request merged**